apps/macos/package.json

```
@@ -15,12 +15,12 @@
     "lint": "fluentui-scripts eslint",
     "start": "react-native start",
     "test": "fluentui-scripts jest",
-    "macos": "react-native run-macos --scheme ReactTestApp --project-path src"
+    "macos": "react-native run-macos --scheme FluentTester --project-path src"
   },
   "dependencies": {
-    "@fluentui-react-native/tester": "^0.13.8",
-    "react-native-macos": "^0.62.0-0",
-    "react-native-test-app": "^0.2.1"
+    "@fluentui-react-native/tester": "^0.13.9",
+    "react-native-macos": "0.63.1",
```

Should we use latest 0.63?
```suggestion
    "react-native-macos": "0.63.2",
```